.eslintrc.json

```diff
@@ -1,24 +1,11 @@
-{
-  "root": true,
-  "extends": [
-    "airbnb-base",
-    "airbnb-base",
-    "prettier",
-    "airbnb/hooks",
-    "plugin:mocha/recommended",
-    "plugin:jest/recommended",
-    "prettier",
-    "plugin:prettier/recommended"
-  ],
-  "plugins": ["node", "prettier", "eslint-plugin-node", "jest"],
+{"root": true,
+  "extends": "airbnb-base",
   "env": {
     "node": true,
     "es6": true,
-    "mocha": true,
-    "jest": true
+    "mocha": true
   },
   "rules": {
-    "prettier/prettier": "error",
```

Comment:
i think this has to be an error or at least a warning, as it will be used by the pre-commit hook